Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:3.0.6
+WORKDIR /usr/src/app
+EXPOSE 3000
```

Comment:
Should we expose and use a port other than the default `3000`? Same question for the ASIS PR.

Reply:
Yes, I think it's better to use unique ports like in the docker-compose file to lower the risk of port conflicts.